src/EditorFeatures/CSharpTest/CodeActions/InlineTemporary/InlineTemporaryTests.cs

```diff
@@ -4459,7 +4457,7 @@ class C
 {
     void M()
     {
-        var t = (where: 1 + 2, 3);
+        var t = (@where: 1 + 2, 3);
```

Comment:
this is fine.  the user wrote the variable name with `@` and we just preserve that.

Reply:
This seems more correct to me as well, since `@int` would probably have to keep the @ otherwise it won't compile.